osgp/platform/osgp-secret-management/pom.xml

```diff
@@ -0,0 +1,192 @@
+<!--
+
+    Copyright 2020 Smart Society Services B.V.
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>osgp-secret-management</artifactId>
+  <name>osgp-secret-management</name>
+  <packaging>war</packaging>
+  <description>Secret management module.</description>
+
+  <parent>
+    <groupId>org.opensmartgridplatform</groupId>
+    <artifactId>parent-platform</artifactId>
+    <version>5.2.0-SNAPSHOT</version>
+    <relativePath>../parent-platform/pom.xml</relativePath>
+  </parent>
+
+  <properties>
+    <java.version>1.8</java.version>
+    <spring.boot.version>2.2.2.RELEASE</spring.boot.version>
+    <jaxb.plugin.version>2.5.0</jaxb.plugin.version>
+    <build.helper.maven.plugin.version>3.0.0</build.helper.maven.plugin.version>
+    <apache.ws.xlmschema.version>2.0.1</apache.ws.xlmschema.version>
+    <apache.jms.api.version>2.0.1</apache.jms.api.version>
+    <logback.spring.extensions.version>0.1.5</logback.spring.extensions.version>
+    <wsdl4j.version>1.6.3</wsdl4j.version>
+    <maven.failsafe.plugin.version>3.0.0-M4</maven.failsafe.plugin.version>
+    <spring.boot.maven.plugin.version>2.2.2.RELEASE</spring.boot.maven.plugin.version>
+    <osgp-ws-secret-management-version>${project.parent.version}</osgp-ws-secret-management-version>
+    <com.h2database>1.4.200</com.h2database>
+    <failOnMissingWebXml>false</failOnMissingWebXml>
+  </properties>
+
+  <build>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=xml%3AIndentCheck)